frontend/sw360-portlet/src/main/webapp/html/projects/includes/attachmentUsagesRows.jspf

```diff
@@ -39,6 +39,11 @@
                         value="${projectLink.name} ${projectLink.version}" maxChar="60"
                 /></a>
             </td>
+            <td>
+            	<sw360:DisplayEnum value="${projectLink.relation}"/>
+            </td>
+            <td>
+            </td>
```

Comment:
Could you please remove the extra <td> if they are not significant here.

Reply:
removed the additional table data tag.